### DISCUSSION TITLE
EmberZNet upgrade on Sonoff Zigbee Bridge - do I have to re-pair all devices?

### DISCUSSION THREAD

**Discussion by AramidX** [arendst/Tasmota#11063] [Q&A] [answered]
Hi All,

Sorry, I could not find any answer to this here, on the FAQs, or via Google, so I got to ask.  I saw that a new version of the EmberZNet NCP UART EZSP firmware (6.7.8) is now available and hopefully fixed the battery drain issues on Ikea switches.  My current Sonoff bridge is on 6.7.6, and was wondering if upgrading to the new version will require me to re-pair all devices.

Thanks!

-A

**Comment by s-hadinger** [comment, marked as answer]
No it doesn't need to repair devices. But you may need to power off on some devices like bulbs or plugs after the upgrade

**Comment by mediastorm2000** [comment]
Still seeing some battery issues with the Tradfri 5 button remote and 2 button remote in particular (6.7.8). I have un-paired and re-paired (joined) the devices back to the network - any suggestions on what to try next? These devices work fine on the Tradfri bridge without eating the batteries - not so much on the Sonoff. Seems inconsistent - added two devices and they did fine for several days. Added a few more devices and then the original devices drained in less than 24 hours. Happy to provide any details/data I can to help towards the issue.

**Reply by mediastorm2000** [reply]
Here is an example - the ones in yellow got new batteries yesterday - all of these devices have been removed from the network and re-joined after the 6.7.8 firmware was applied.

![image](https://user-images.githubusercontent.com/66156358/110532676-c15d1800-80e2-11eb-8998-003e7038df83.png)

**Reply by mediastorm2000** [reply]
As anticipated, the Tradfri 5 button remote and 2 button remote shown above have drained to the point of falling off the network. 

The Symfonisk seems to be holding steady so far - will continue monitoring that one.

Measured voltage on the depleted batteries was down to 2.2-2.3v from the original 3.36 less than 24 hours ago.

![image](https://user-images.githubusercontent.com/66156358/110581982-1d4e8d80-8131-11eb-98c0-e0755fbee56d.png)
